sql/core.go

```diff
@@ -260,6 +263,7 @@ type IndexAlterableTable interface {
 
 // InsertableTable is a table that can process insertion of new rows.
 type InsertableTable interface {
+	Table
```

Comment:
Seems a bit weird having `InsertableTable` implement `Table` but the other `-ableTable`s do not, but I guess it's also nbd

Reply:
I'll make this consistent. I did this for a dumb reason (debugging support), not attached to it.